modules/utils/keyboard_util.py

```diff
@@ -96,7 +96,11 @@ def get_vote_kb(published_post: PublishedPost = None) -> InlineKeyboardMarkup:
                                                 callback_data=f"meme_vote,{reaction_id}"))
         keyboard.append(new_row)
     # the last button in the last row will be the report button
-    keyboard[-1].append(InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot,"))
+    if config_map['meme']['report']:
+        if len(keyboard) > 0:
```

Comment:
```suggestion
        if len(keyboard):
```

Reply:
Pylint suggestion:
`Do not use "len(SEQUENCE)" without comparison to determine if a sequence is empty pylint(len-as-condition)`

Reply:
On JS is really common to use this without a comparison but because there we use arr.length to check it and not a function, so sorry :P